### PR TITLE
chore(core): add ability to set up a device with SLIP39 "single"

### PR DESCRIPTION
**Pull request by ibz**

### CODE REVIEW

File: core/.changelog.d/3868.added [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Added flag for setting up device using SLIP39 "single".
+Renamed flag used for setting up device using BIP39 to `basic`.
```
Comment (matejcik) [inline core/.changelog.d/3868.added:2]: these should be two changelog entries, one for `added`, other for `incompatible`

**Comment by matejcik** [comment]
it's a chore for `python` not `core`

**Comment by matejcik** [comment]
honestly maybe more of a `feat`

File: core/.changelog.d/3868.added [hunk 2]
```diff
@@ -0,0 +1 @@
+Added flag for setting up device using SLIP39 "single".
```
Comment (matejcik) [inline core/.changelog.d/3868.added:1]: ...and of course the changelogs go to `python/`

**Review by mmilata — APPROVED** [review]

**Comment by STew790** [comment]
**QA OK**
Done together with Suite https://www.notion.so/satoshilabs/Creating-new-multishare-backup-testing-activity-ab3a703401f344dab67537ecb327acf0

**Info:**
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: Trezor T3T1 2.7.2 regular (revision da75d8f4b67410b40a9cfd2954d183d81dd6e8e8)
 - Transport: BridgeTransport 2.0.33

**Pull request merged**